index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Katherine Paulino Lucas</title>
+    <link href='https://fonts.googleapis.com/css?family=Satisfy' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
+    <link href="vendor/normalize.css" type="text/css" rel="stylesheet">
+    <link href="vendor/hover.css" type="text/css" rel="stylesheet">
+    <link href="styles/style.css" type="text/css" rel="stylesheet">
+  </head>
+  <body align='center'>
+    <header>
+    <h1>Katherine Paulino Lucas</h1>
+    </header>
+    <nav>
+      <ul class='nav'>
+        <li><a href='#about' class='hvr-grow'>about</a><li>
+        <li><a href='#projects' class='hvr-grow'>projects</a><li>
+        <li><a href='#social' class='hvr-grow'>social</a><li>
+        <li><a href='mailto:katherinelucas@gmail.com' class='hvr-grow'>contact</a><li>
+      </ul>
+    </nav>
+    <main>
+      <section id='about'>
+        <p class='container'>
+        <img id='aboutpic' src='img/kate2.png' height=150 width=150>
+        <span>Lorizzle ipsizzle own yo' sit go to hizzle, consectetuer adipiscing doggy. My shizz crazy velizzle, ghetto volutpizzle, cool quizzle, away vizzle, sure. Gangster eget tortor. Funky fresh erizzle. Fusce at pizzle dapibus turpis pimpin' tempor. Maurizzle pellentesque nibh izzle turpizzle. Pimpin' in tortizzle. Pellentesque fizzle rhoncus nisi. Daahng dawg hac habitasse platea dictumst. Mah nizzle away. Curabitur tellus urna, pretizzle da bomb, mattizzle go to hizzle, i'm in the shizzle ma nizzle, nunc. Yippiyo suscipizzle. Izzle bizzle velit sed check out this.</span>
+        </p>
+      </section>
+      <section id='projects'>
+        <h2>Projects</h2>
+          <section id="projects">
```

Comment:
This section is indented too far.  It should have the same level of indention as the h2.